### PR TITLE
Add Test for deleteNotificationsByTopic

### PR DESCRIPTION
**Pull request by O-Susanoo**

### CODE REVIEW

File: services/user/test/user.test.js [hunk 1]
```diff
@@ -510,41 +510,66 @@ describe('User Class', () => {
...
-      const res = await getChangelog()
+    async function fireDeleteNotificationsByTopicEvent(requestedUserId, requestingUserId, topic) {
+      const event = deleteNotificationsByTopicEvent(requestedUserId, requestingUserId, topic)
+      const res = await deleteNotificationsByTopic(event, {})
 
       return {
         res,
         body: tryParseJSON(res.body)
       }
     }
```
Comment (DanielSeehausen) [inline services/user/test/user.test.js]: because tests tend to grow overtime, this variable is a little too global, as its in the top level of the function's describe block.

some possible solutions:
consider either nesting it in another describe block:
```js
describe('when the topic is changelog', () => 
  const TOPIC = 'changelog'
  test(...)
  test(...)
})
```

or make the `topic` `test` scope repeatedly!

File: services/user/test/user.test.js [hunk 2]
```diff
@@ -510,41 +510,66 @@ describe('User Class', () => {
...
 
-    test('should be called with the right arguments and return a changelog pull request', async() => {
+    const topic = 'changelog'
 
-      process.env.GITHUB_ACCESS_TOKEN='fake token'
+    test('should delete changelog topic notifications from the user', async() => {
+      const notification = generateNotification({topic : topic})
```
Comment (DanielSeehausen) [inline services/user/test/user.test.js]:
```js
const notification = generateNotification({topic : topic})
// is equal to
const notification = generateNotification({ topic })
```

**Comment by DanielSeehausen** [comment]
looks great! nothing blocking!

tests are A+

the auto formatting you are using looks good, please keep it to only the stuff the PR is touching, though. This is strictly to make PR reviewing more effective. There is a lot of good data to suggest the more focused a PR is, the better developers are at effectively reading it and catching bugs.

TL;DR:
10 lines of code === 1 bug found
200 lines of code === "Looks great"

**Comment by O-Susanoo** [comment]
> looks great! nothing blocking!
> 
> tests are A+
> 
> the auto formatting you are using looks good, please keep it to only the stuff the PR is touching, though. This is strictly to make PR reviewing more effective. There is a lot of good data to suggest the more focused a PR is, the better developers are at effectively reading it and catching bugs.
> 
> TL;DR:
> 10 lines of code === 1 bug found
> 200 lines of code === "Looks great"

To be honest this the biggest issue i'm facing , we really need to talk about formatting either we make a commit that will format everything or figure out something else
